mash_client/cli/job/__init__.py

```diff
@@ -84,9 +95,143 @@ def get(context, job_id):
             action='get'
         )
 
+        if not show_data:
+            with suppress(KeyError):
+                del result['data']
+
         echo_dict(result, config_data['no_color'])
 
 
+@click.command()
+@click.option(
+    '--job-id',
+    type=click.UUID,
+    required=True,
+    help='The UUID of the job to be removed '
+         'from the MASH server pipeline.'
+)
+@click.pass_context
+def status(context, job_id):
+    """
+    Get basic status for a job in the MASH server pipeline.
+    """
+    config_data = get_config(context.obj)
+
+    with handle_errors(config_data['log_level'], config_data['no_color']):
+        result = handle_request_with_token(
+            config_data,
+            '/jobs/{0}'.format(job_id),
+            action='get'
+        )
+
+        status_info = {'state': result['state']}
+
+        if result['state'] == 'running' and 'current_service' in result:
+            status_info['current_service'] = result['current_service']
+
+        echo_dict(status_info, config_data['no_color'])
+
+
+@click.command()
+@click.option(
+    '--job-id',
+    type=click.UUID,
+    required=True,
+    help='The UUID of the job to be removed '
```

Comment:
In the context of status the job does not get removed. This should probably read "The UUID of the job for the status query."

Reply:
Copy/paste typo :-1: